### PR TITLE
Unwrap ProofBytes structure in proof verification info structs

### PR DESCRIPTION
**Pull request by anorth**
These structures used to have other fields, but now don't.

FYI @dignifiedquire, please alert if we'll need to revert this because there will be more fields again.

### CODE REVIEW

File: actors/abi/sector.go [hunk 1]
```diff
@@ -32,25 +32,26 @@ type SectorWeight = big.Int
...
-	RegisteredProof_StackedDRG32GiBPoSt    = RegisteredProof(4)
-	RegisteredProof_StackedDRG1KiBSeal     = RegisteredProof(5)
-	RegisteredProof_StackedDRG1KiBPoSt     = RegisteredProof(6)
-	RegisteredProof_StackedDRG16MiBSeal    = RegisteredProof(7)
-	RegisteredProof_StackedDRG16MiBPoSt    = RegisteredProof(8)
-	RegisteredProof_StackedDRG256MiBSeal   = RegisteredProof(9)
-	RegisteredProof_StackedDRG256MiBPoSt   = RegisteredProof(10)
-	RegisteredProof_StackedDRG1GiBSeal     = RegisteredProof(11)
-	RegisteredProof_StackedDRG1GiBPoSt     = RegisteredProof(12)
+	RegisteredProofWinStackedDRG32GiBSeal = RegisteredProof(1)
```
Comment (dignifiedquire) [inline actors/abi/sector.go]: winstacked is not a valid algorithm anymore, why is this still here?

File: actors/abi/sector.go [hunk 2]
```diff
@@ -32,25 +32,26 @@ type SectorWeight = big.Int
...
+	RegisteredProofWinStackedDRG32GiBPoSt = RegisteredProof(2)
+	RegisteredProofStackedDRG32GiBSeal    = RegisteredProof(3)
+	RegisteredProofStackedDRG32GiBPoSt    = RegisteredProof(4)
+	RegisteredProofStackedDRG1KiBSeal     = RegisteredProof(5)
```
Comment (dignifiedquire) [inline actors/abi/sector.go]: if this is spec code, the non spec compliant sectors should be probably marked as such,

File: actors/abi/sector.go (hunk 1 above)
Comment (anorth) [reply actors/abi/sector.go]: Because no-one responsible who knows that updated the spec.

File: actors/abi/sector.go (hunk 2 above)
Comment (anorth) [reply actors/abi/sector.go]: Indeed. Please tell us what they are (preferably by PR).

**Comment by anorth** [comment]
We can't unwrap them after all.

**Pull request closed without merging**